### ISSUE TITLE
"comment out until we have type coercion"

### ISSUE THREAD

**Issue by jbrockmendel** [statsmodels/statsmodels#4324]
There are a handful of tests in test_tools that are commented out, with a comment suggesting they are waiting on type coercion.  Is there a reference for this somewhere?  It would be easy for these to be forgotten when type coercion is eventually implemented (and it wouldn't surprise me if it already has been)

**Comment by jbrockmendel** [comment]
Closing in favor of #5145.
